application/modules/quotes/views/modal_copy_quote.php

```diff
@@ -56,14 +56,14 @@ function (data) {
             </div>
 
             <div class="form-group has-feedback">
-                <label for="quote_date_created">
+                <label for="quote_date_created_modal">
                     <?php _trans('quote_date'); ?>
                 </label>
 
                 <div class="input-group">
-                    <input name="quote_date_created" id="quote_date_created"
+                    <input name="quote_date_created_modal" id="quote_date_created_modal"
                            class="form-control datepicker"
-                           value="<?php echo date_from_mysql($quote->quote_date_created, true); ?>">
+                           value="<?php echo date_from_mysql(date('Y-m-d', time()), true); ?>">
```

Comment:
I think this should be discussed. Here we are changing the original behavior which was to suggest as creation date also a date in the past. I believe this is over the changes expected with the resolution of the issue. I suggest the change is reverted.

Reply:
@naui95, I agree that this should be bediscussed. 
I have no problem reverting this change but...
If I find an "old quote/invoice" with the same items as the one I need then I will use this copy functionality provided I only need to change some limited items. 
Anyway, I will usually always change the old quotation/invoice date to the current date. 
In my opinion it makes no sense to keep that old date.
Just my 2 cents.
> I leave it up to you to decide.

Reply:
@Verony-makesIT you actually have a point (or more than one). The behavior you are suggesting is also followed when copying invoices and I suppose is even the expected one.
https://github.com/InvoicePlane/InvoicePlane/blob/bd22ac6f06dd9860d1f48278eaeff6c90674a026/application/modules/invoices/views/modal_copy_invoice.php#L70
My comment is in fact narrow minded, because in principle a PR should only solve the bug/enhancement/etc. to which it is related. This PR solves multiple bugs (thank you!), not all referenced by the issue, and when I went through your code this change did not look as referenced to a bug to me and...therefore the revert suggestion.

Let's see what @nielsdrost7 thinks about merging this change. I believe that it might be wise to keep it in, as @Verony-makesIT pointed out.

Reply:
@mheiduk Is this another one of those situations where the date suddenly changes? Just wondering.